zeppelin-web/src/app/notebook/paragraph/result/result.controller.js

```diff
@@ -263,7 +263,9 @@ function ResultCtrl($scope, $rootScope, $route, $window, $routeParams, $location
       renderApp(app);
     } else {
       if (type === 'TABLE') {
-        $scope.renderGraph($scope.graphMode, refresh);
+        $timeout(function() {
+          $scope.renderGraph($scope.graphMode, refresh);
```

Comment:
@prabhjyotsingh 

Can we modify `renderGraph` function directly? because this function already has some timeout logic. But it seems that it doens't work. 

https://github.com/prabhjyotsingh/zeppelin/blob/ca1ea2e58d0e011476b5c83e3ae94a2b27668a73/zeppelin-web/src/app/notebook/paragraph/result/result.controller.js#L382